### ISSUE TITLE
cannot refer to `dateFilterCondition`

### ISSUE THREAD

**Issue by boogiebug** [ghiscoding/slickgrid-universal#1077]
### Describe the bug

I'm creating a custom server service. and cannot refer the exported function `executeDateFilterCondition` in `packages/common/src/filter-conditions/dateFilterCondition.ts`.

I suppose that this package (dateFilterCondition) should be als exported in `packages/common/src/filter-conditions` as others in the same folder but it's missing.

Current content of the file `filter-conditions/index.ts` is:
 
```ts
export * from './stringFilterCondition';
export * from './objectFilterCondition';
export * from './numberFilterCondition';
export * from './filterUtilities';
export * from './filterConditions.index';
export * from './filterConditionProcesses';
export * from './collectionSearchFilterCondition';
export * from './booleanFilterCondition';
```
 I suppose the the following line is missing?
 
 ```ts
 export * from './dateFilterCondition';
 ```

### Reproduction

This is not a run-time bug, but the compile time bug. To reproduce this bug, simply refer to any exported function from `dateFilterCondition` module in the source code and the compile errors will be appeared.

### Which Framework are you using?

Angular

### Environment Info

```shell
| Executable          | Version |
| ------------------- | ------- |
| Slickgrid-Universal | 3.1     |
| TypeScript          | 4.9     |
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/slickgrid-universal/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [Wikis](https://github.com/ghiscoding/slickgrid-universal/wiki).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/slickgrid-universal/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/slickgrid-universal/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
I'm not sure why you would want to extend these filter conditions, I mean I'm curious to see why. So yes, it looks like it is missing and a Pull Request is certainly welcome. I'm expecting to push a new release in the next couple days so it would be nice if you have the time to create a PR. Thanks

**Comment by ghiscoding** [comment]
closed by #1078, thanks again
